frontend/src/app/page.tsx

```diff
@@ -1,13 +1,275 @@
-import React from 'react';
+'use client';
+import AccordionItem from '@/components/Accordion';
+import FacebookIcon from '@/components/icon/facebook-icon';
+import InstagramIcon from '@/components/icon/instagram-icon';
+import TwitterIcon from '@/components/icon/twitter-icon';
+import ImageBackground from '@/components/ImageBackground';
+import LandingPageHeader from '@/components/LandingPageHeader';
+import Image from 'next/image';
+import { GoDotFill } from 'react-icons/go';
+
+const faqs = [
+  {
+    title: 'What is Youcompare ?',
+    content:
+      'Youcompare is a platform that helps you find the best tools for your business. We provide comprehensive comparisons of cloud services, databases, CMS platforms, and project management tools, so you can make informed decisions.',
+  },
+  {
+    title: 'How does the platform assist small business ?',
+    content:
+      'Our platform is designed to help small businesses find the right tools for their needs. We provide detailed comparisons of various services, so you can easily find the best fit for your business.',
+  },
+  {
+    title: 'Is the platform free for users ?',
+    content:
+      'Yes, our platform is completely free for users. You can explore and compare different tools without any cost or commitment.',
+  },
+  {
+    title: 'Is the platform up to date with prices ?',
+    content:
+      'Yes, we regularly update our platform with the latest prices and features of various tools.',
+  },
+];
+
+const allFeatures = [
```

Comment:
Create a folder name utils/constants.ts and move this in there and export it

Reply:
resolved